tests/fixtures/hub.py

```diff
@@ -15,9 +15,17 @@
 
 CI_HUB_ENDPOINT = "https://hub-ci.huggingface.co"
 CI_HUB_DATASETS_URL = CI_HUB_ENDPOINT + "/datasets/{repo_id}/resolve/{revision}/{path}"
+CI_HFH_HUGGINGFACE_CO_URL_TEMPLATE = CI_HUB_ENDPOINT + "/{repo_id}/resolve/{revision}/{filename}"
 CI_HUB_TOKEN_PATH = Path("~/.huggingface/hub_ci_token").expanduser()
 
 
+@pytest.fixture
+def ci_hfh_hf_hub_url(monkeypatch):
+    monkeypatch.setattr(
+        "huggingface_hub.file_download.HUGGINGFACE_CO_URL_TEMPLATE", CI_HFH_HUGGINGFACE_CO_URL_TEMPLATE
+    )
+
```

Comment:
@Wauplin, just please note that in our CI currently we need to do the patching in `huggingface_hub.file_download` because doing it in `huggingface_hub.constants` has no effect: at import time, the constant `HUGGINGFACE_CO_URL_TEMPLATE` was copied from the submodule `constants` to `file_download`.

Reply:
Understood, thanks for the reminder. I created an issue on `huggingface_hub` to keep track of this. 
https://github.com/huggingface/huggingface_hub/issues/1172